test/test_custom_ops.py

```diff
@@ -678,6 +680,134 @@ def foo(x: Tensor, y: int) -> Tensor:
 
             infer_schema(foo, mutated_args={"y"})
 
+    def test_blackbox_basic(self):
+        @def_blackbox(mutated_args=())
+        def f(x: Tensor, y: float) -> Tensor:
+            return x + y
```

Comment:
These examples are awkward because there isn't actually any reason for this operator to be black box